docs/en/connector-v2/sink/Hive.md

```diff
@@ -25,21 +25,24 @@ By default, we use 2PC commit to ensure `exactly-once`
   - [x] parquet
   - [x] orc
   - [x] json
+  - [x] hive_text : support the text storage format of hive collection data type, array, map, struct.
```

Comment:
`hive_text` is not a file format, can not put it here. You can use example to tell uses how to use it.